Source/SwiftLintFramework/Rules/Lint/UseCaseExposedFunctionsRule.swift

```diff
@@ -0,0 +1,126 @@
+import SourceKittenFramework
+
+struct UseCaseExposedFunctionsRule: ASTRule, ConfigurationProviderRule {
+    var configuration = SeverityConfiguration(.error)
```

Comment:
Should the severity be a warning? If we use this in our project, this will fail all our logic classes and it will prevent us from building the app.